dotnet/src/VectorDataIntegrationTests/SqlServerIntegrationTests/Filter/SqlServerBasicFilterTests.cs

```diff
@@ -0,0 +1,81 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using Microsoft.Extensions.VectorData;
+using SqlServerIntegrationTests.Support;
+using VectorDataSpecificationTests.Filter;
+using VectorDataSpecificationTests.Support;
+using Xunit;
+using Xunit.Sdk;
+
+namespace SqlServerIntegrationTests.Filter;
+
+public class SqlServerBasicFilterTests(SqlServerBasicFilterTests.Fixture fixture)
+    : BasicFilterTests<int>(fixture), IClassFixture<SqlServerBasicFilterTests.Fixture>
+{
+    // "SELECT * FROM MyTable WHERE BooleanColumn = 1;" is fine
+    // "SELECT * FROM MyTable WHERE BooleanColumn;" is not
+    // TODO adsitnik: get it to work anyway
```

Comment:
Yeah, this is a "particularity" of T-SQL which we should handle in the translator.